test/routes/index.spec.unit.ts

```diff
@@ -0,0 +1,24 @@
+import app from "../../src/app";
+import * as request from "supertest";
+
+describe("Basic URL Tests", () => {
+
+  it("should find start page", async () => {
+    const response = await request(app)
+      .get("/promise-to-file");
+    expect(response.status).toEqual(200);
```

Comment:
how do you know it's returning the correct page?  can you check page title or some identifying string in response.text (i think)